src/ndi-source.cpp

```diff
@@ -491,71 +535,82 @@ void *ndi_source_thread(void *data)
 					NDIlib_recv_color_format_fastest;
 			blog(LOG_INFO,
 			     "[DistroAV] ndi_source_thread: '%s' latency changed; Setting recv_desc.color_format='%d'",
-			     obs_source_ndi_receiver_name,
+			     obs_source_name, //
 			     recv_desc.color_format);
 		}
+
 		if (config_most_recent.framesync_enabled !=
 		    config_last_used.framesync_enabled) {
 			config_last_used.framesync_enabled =
 				config_most_recent.framesync_enabled;
 
-			reset_recv_desc = &recv_desc;
+			reset_ndi_receiver = true;
+
 			blog(LOG_INFO,
 			     "[DistroAV] ndi_source_thread: '%s' framesync changed to %s",
-			     obs_source_ndi_receiver_name,
+			     obs_source_name, //
```

Comment:
is the "// " expected here ?

Reply:
Yes; it forces the similar above and below lines of code all consistently line wrap nearly identically.